### ISSUE TITLE
Error on step 2 of Deploy Azure Resources: malformed node or string: <_ast.Name object at 0x05260BB0>

### ISSUE THREAD

**Issue by jyork13** [mspnp/architecture-center#767]
I kept getting this error during step 2 of Deploy Azure Resources.  I finally realized it was because of the quotes around the parameter names.  Once I removed the quotes from the parameter names it started working. For example --parameters "dwAdminLogin"="adminuser"... should be --parameters dwAdminUser="adminuser"...

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c337b22b-b387-5eea-9b16-656885eae759
* Version Independent ID: 6c1ad9b6-fdf4-f23a-20dc-118f262d0a8c
* Content: [Automated enterprise BI with SQL Data Warehouse and Azure Data Factory](https://docs.microsoft.com/en-us/azure/architecture/reference-architectures/data/enterprise-bi-adf#deploy-the-solution)
* Content Source: [docs/reference-architectures/data/enterprise-bi-adf.md](https://github.com/mspnp/architecture-center/blob/master/docs/reference-architectures/data/enterprise-bi-adf.md)
* Service: **guidance**
* GitHub Login: @MikeWasson
* Microsoft Alias: **pnp**

**Comment by MikeWasson** [comment]
Thanks for reporting this. What shell were you using? I've seen some differences in how arguments are parsed on windows vs bash.

**Comment by jyork13** [comment]
I'm using Powershell in windows 8.1.

Jim York<https://www.linkedin.com/in/yorkanalytics>
York Analytics
[yorkA-6 no shadow - 72 dpi]

**Comment by MikeWasson** [comment]
Thanks for reporting this. I'm closing the issue because you are unblocked, but I opened a separate issue in our internal backlog to review how we write these deployment steps.
